packer/sagebio/sagebio-base-ubuntu-bionic-LATEST/playbook.yaml

```diff
@@ -17,13 +17,18 @@
           - "curl"
         state: present
 
-    # awscli is on aws linux2 distro by default
-    - name: install awscli
+    - name: Prep snap classic
+      shell: "[ -x /snap ] || yum install -y epel-release snapd && systemctl enable --now snapd.socket && ln -s /var/lib/snapd/snap /snap || echo ''" #install via yum on CentOS if DNE
+      register: snap_installed
+
+    # awscli and amazon-ssm-agent are on aws linux2 by default
+    - name: Install awscli and amazon-ssm-agent if available in Snappy
```

Comment:
you are going to install snappy. that's a fact.    nothing in this file validates that those tools are available in snap repo.  so maybe just `Install aws tools`?

Reply:
agreed